test/blackbox-tests/test-cases/melange/aliases.t

```diff
@@ -43,6 +43,29 @@ Default alias melange works
   $ node _build/default/output2/main2.js
   hello
 
+Dune default alias works
+
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (target output)
+  >  (modules main))
```

Comment:
Emit stdlib false on these too

Reply:
just to confirm: do you mean new test cases are desired?

Reply:
ah, I guess it's related to https://github.com/ocaml/dune/pull/7926#discussion_r1225147391 so I believe you mean just adding `(emit_stdlib false)` to the new test cases

Reply:
reflected in 7c49c1d7027915cc9aff8d3a76fa58081ba84ecd